view/adminhtml/templates/client_iframe.phtml

```diff
@@ -18,7 +18,7 @@
     var eventBinding = {};
     eventBinding[Protect.EventName.ORDER_DETAIL_NAME_CLICK] = navigateToMagentoOrderDetails;
     var clientConfig = new Protect.ClientConfig({
-      accessToken: '<?= $block->config->getAccessToken(); ?>',
+      accessToken: '<?= $block->config->getAccessToken(null); ?>',
```

Comment:
Could probably leave it as it was and make `null` the default so you don't need to update all these calls.